src/subscript/sector2fluxnum/fluxfile_obj.py

```diff
@@ -0,0 +1,457 @@
+            kw_temp[j] = flux_fine[i][j]
+
+        kw_temp.fwrite(fortio)  # Writes to file succesivly
+
+    # Prop elements
+    nPropElements = len(flux_fine[10])
+
+    newFluxSize = len(flux_coarse)
+
+    # Manipulating existing blocks in the fine grid
+    # Importing data from the coarse grid
+    for i in range(nCommonElements, newFluxSize):
+
+        #    print "Reading element %s" % flux_coarse[i].header[0]
+
+        # Copies directly from coarse grid to fine grid.
+        # Not related to grid cells
+        if flux_coarse[i].header[0] == "ITIME":
+            kw_temp = EclKW(
+                flux_coarse[i].header[0], flux_coarse[i].header[1], flux_coarse[i].type
+            )
+
+            kw_source = flux_coarse[i]
+
+            for j in range(len(kw_temp)):
+                kw_temp[j] = flux_coarse[i][j]
+
+        elif flux_coarse[i].header[0] == "DTIME":
+            kw_temp = EclKW(
+                flux_coarse[i].header[0], flux_coarse[i].header[1], flux_coarse[i].type
+            )
+
+            kw_source = flux_coarse[i]
+
+            for j in range(len(kw_temp)):
+                kw_temp[j] = flux_coarse[i][j]
+
+        elif flux_coarse[i].header[0] == "WELLNAME":
+            kw_temp = EclKW(
+                flux_coarse[i].header[0], flux_coarse[i].header[1], flux_coarse[i].type
+            )
+
+            kw_source = flux_coarse[i]
+
+            for j in range(len(kw_temp)):
+                kw_temp[j] = flux_coarse[i][j]
+
+        elif flux_coarse[i].header[0] == "WELLFLOW":
+            kw_temp = EclKW(
+                flux_coarse[i].header[0], flux_coarse[i].header[1], flux_coarse[i].type
+            )
+
+            kw_source = flux_coarse[i]
+
+            for j in range(len(kw_temp)):
+                kw_temp[j] = flux_coarse[i][j]
+
+            # Maps property data from coarse to fine grid.
+            # Related to grid cells.
+        else:
+            kw_temp = EclKW(
+                flux_coarse[i].header[0], nPropElements, flux_coarse[i].type
+            )
+
+            kw_source = flux_coarse[i]
+
+            for j in range(nPropElements):
+                kw_temp[j] = kw_source[mapping[j]]
+                # kw_temp[j] = flux_coarse[i][f_c_mapping[j]]
+                # kw_temp[j] = flux_coarse[i][j]
+
+        print("Writing element %s" % flux_coarse[i].header[0])
+        kw_temp.fwrite(fortio)
+
+
+def write_new_fluxfile_from_rst(destFlux, sourceGrid, sourceRST, mapping, fortio):
+    """
+    Populates a templated .FLUX file with full field data from a RESTART file.
+
+    @destFlux: Template FLUX file to be populated
+    @sourceFlux: Template FLUX file from full field grid (Not used)
+    @sourceRST: RESTART data from full field simulation. Used to populate @destFlux
+    @mapping: Map from refined to coarse index
+    @fortio: File stream for unformated data.
+
+    The population of the FLUX file is sensitive to the format of the template file.
```

Comment:
My experience is that ECL is pretty back compatible for obvious reasons. If the template of the FLUX files is changed it can cause fundamental errors as properties read in are interpreted as something else. We can take the discussion.